cookbook/configuration/pdo_session_storage.rst

```diff
@@ -172,10 +229,10 @@ following (MySQL):
 .. code-block:: sql
 
     CREATE TABLE `session` (
-        `session_id` VARBINARY(128) NOT NULL PRIMARY KEY,
-        `session_data` BLOB NOT NULL,
-        `session_time` INTEGER UNSIGNED NOT NULL,
-        `session_lifetime` MEDIUMINT NOT NULL
+        `sess_id` VARBINARY(128) NOT NULL PRIMARY KEY,
+        `sess_data` BLOB NOT NULL,
```

Comment:
But don't you think we should say explicitly the bytes allowed by the BLOB type? Otherwise it may be as confusing as when some web forms tells you *"this is too long"* but it doesn't tell you which is the limit.